website/docs/quick-start-guide.md

```diff
@@ -216,9 +237,33 @@ can generate sample inserts and updates based on the the sample trip schema [her
 
 ## Create Table
 
+Before we go further, let us go over few terminologies: 
+
+- **Table types**
+
+  Hudi supports two different table types, namely Copy-On-Write (COW) and Merge-On-Read (MOR). Users can choose either 
+of these table types depending on their workload and SLA requirements. You can read more about different 
+  table types [here](/docs/next/table_types/).
+
+- **Partitioned & Non-Partitioned tables**
+
+  Users can create a partitioned table or a non-partitioned table with Apache Hudi. Partitioning can help with 
+  reducing query run times. For quick start purpose, we will go with partitioned table. 
+
+- **Primary key and Hudi table**
+
+  Optionally users can choose to create a Primary keyed table. When primary key is set for a given table,
```

Comment:
lets not create more artificial distinctions for the tables here. Hudi tables always have a key, its a matter of whether its user defined or system generated.